overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -8748,7 +8748,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Stunning Stones",
-          "desc:8": "Craft at least 1 Infusion Cost Stone and 1 Infusion Speed Stone. By placing them below the main legs of the Runic Matrix multiblock, you can either reduce the cost of the crafts or speed them up! Your choice... You can use at most 4 of these, obviously."
+          "desc:8": "Craft at least 1 Infusion Cost Stone and 1 Infusion Speed Stone. By placing them below the main legs of the Runic Matrix multiblock, you can either reduce the cost of the crafts or speed them up! Your choice... You can use at most 4 of these, obviously. Note that you should only use one type of these per altar, otherwise they conflict with each other and you get no benefits."
```

Comment:
according to this reddit comment (couldn't find anything else on this, didn't want to figure out how to decompile jars), this change is incorrect

https://www.reddit.com/r/Thaumcraft/comments/g8larb/comment/fr261wd/

Reply:
Hmm, good to know.